### ISSUE TITLE
How to remove the labels at the sides of the document? 

### ISSUE THREAD

**Issue by Andresfgomez970** [martinhelso/mnfrontpage#1]
Hi, 

First of all, thank you very much for this template. I am using it for a compendium of problems in quantum field theory. However, there is one thing I have not been able to modify and I would appreciate if you have any input. 

**The problem that I have**

I do want to use labels in latex equations, but I do not want a label with the name of the equations by the side. I mean: 

The expression
`
\begin{equation}
L = \int_{0}^{a} dx \Bigg[ \frac{\sigma}{2} \bigg(\frac{\partial y}{\partial t}\bigg)^2 - \frac{T}{2} \bigg(\frac{\partial y}{\partial x}\bigg)^2  \Bigg] 
\label{eq:L_y}
\end{equation}
` 

renders into: 

![imagen](https://github.com/uio-latex/mnfrontpage/assets/26715803/49081fc0-c7af-46ba-9529-64b325b11b5d)

So, how to I remove the part at the right that says {eq:L_y}?

**What I have tried**

I did how to the style.sty document a try to see if there was any comment regarding how to deactivate this but was not able to find anything. 

I would really appreciate any input. Thank you so much again for the great template.

**Comment by Andresfgomez970** [comment]
Hi again, 

after looking in more detail I see that the github repo is about the mnfrontpage. However, as this was the only collaborative page I did find about latex documents affiliated with the Faculty of Mathematics and Natural Sciences, University of Oslo. I will pass to explain how I solved the problem: 

style.sty contains the following lines: 

`
\RequirePackage[notref, notcite]{showkeys}               % Labels in margins
\renewcommand{\showkeyslabelformat}[1]
{
    \fbox
    {
        \parbox[t]{0.6\marginparwidth}
        {
            \centering\normalfont\footnotesize\ttfamily
            \path{#1}
        }
    }
}
` 

I did change this to the following and the problem was solved:


`
\RequirePackage[notref, notcite]{}               % Labels in margins
` 

In fact, it is very simple. I did post this with the hope it can save someone some time.
